src/tests/unit_tests/test_vectorstores.py

```diff
@@ -0,0 +1,186 @@
+# Copyright 2024 Google LLC
```

Comment:
We are pending test coverage for methods like
` max_marginal_relevance_search_by_vector`  and others from vectorstores.py , the async methods, etc.

Reply:
Added